ibis/expr/builders.py

```diff
@@ -145,8 +146,8 @@ class WindowBuilder(Builder):
     how: Literal["rows", "range"] = "rows"
     start: Optional[RangeWindowBoundary] = None
     end: Optional[RangeWindowBoundary] = None
-    groupings: VarTuple[Union[str, Resolver, ops.Value]] = ()
-    orderings: VarTuple[Union[str, Resolver, ops.SortKey]] = ()
+    groupings: VarTuple[Union[str, Resolver, Selector, ops.Value]] = ()
+    orderings: VarTuple[Union[str, Resolver, Selector, ops.SortKey]] = ()
```

Comment:
Since there's a bit of noise from dealing with the import cycle from using `Selector` at the top level of `builders.py`, I'm calling out that this is the change that enables use of a `Selector` in the `group_by` and `order_by` window APIs.

We're already calling `Table.bind` on these two fields in the appropriate place, we simply needed to allow them in the signature of this builder to allow them to get passed through the API.